### DISCUSSION TITLE
编译出来的参考文献会把bib文件里的文件全部列出来，怎么修改让其只列我引用过的文献

### DISCUSSION THREAD

**Discussion by yushan-li** [sjtug/SJTUThesis#1011] [提问]
编译出来的参考文献会把bib文件里的文件全部列出来，怎么修改让其只列我引用过的文献；
我用的是sublime配的本地环境编译

**Comment by LogCreative** [comment]
一般只会列出已引用的文献，建议检查一下文档中有没有包含 `\nocite{*}`，将其移除。
